docker/docker_client.go

```diff
@@ -43,18 +42,13 @@ const (
 
 	minimumTokenLifetimeSeconds = 60
 
+	retryAfterDelay = 20 // 20 seconds of delay after the request
```

Comment:
That's too much. For example rate limiting in quay.io applies on seconds base.

I'd start with 2 seconds?

Reply:
2, 4, 8? What should the sequence look like?

Reply:
2, 4, 8, 16, 32? 5 retries

Reply:
@twaugh WDYT?

Reply:
Yes, if there is no Retry-After supplied in the header, the best behaviour would be to back off exponentially like that. 2, 4, 8, 16, 32, fail, seems about right.